### ISSUE TITLE
Just installed vscode plugin...how do I use it?

### ISSUE THREAD

**Issue by dcs3spp** [pierre3/PlantUmlClassDiagramGenerator#38]
Just installed vscode plugin...how do I use it?

Can you please include a link to documentation that details how to use the plugin extension in vscode. I have set the input path in the extension plugins. How do I generate the diagram????

**Comment by p-buddy** [comment]
Hi @dcs3spp -- did you figure it out? I too am wondering!

**Comment by kdmayo** [comment]
Yeah I just installed this too ---- how do you use it?????  There doesn't seem to be any new VS Code menu items

**Comment by OscarKro** [comment]
I found the solution after some searching on the internet. You can execute commands in VS code by pressing ctrl-shift-P. Before I run any command I went to settings, and set the input path of the directory I wanted to have as plant uml code. After that, I dit ctrl-shift-P, typed in "csharp2plantuml.classDiagram" and it seemed to have worked. I placed all UML files in a seperate directory called "plantuml" which it defaults to. But that can also be changed in the settings. All the commands you can currently give are typed in this changelog from the extension 
![image](https://user-images.githubusercontent.com/43294348/116861079-d5188c80-ac02-11eb-89e0-8c0c51b95f4b.png)

**Comment by OscarKro** [comment]
I commented first, but now just made a video on youtube about it. Hope this helps you: [How to use CSharp to PlantUML in VS Code ](https://youtu.be/ErEo-alpS6c)

**Comment by promethyttrium** [comment]
There are no error messages or any kind of output so you can't tell you set the directories correctly when you run that command; all you don't see is the output directory. This extension is completely useless.

**Comment by Jeroen-Sturm** [comment]
If this is still an issue:

In VSCode go to : `View > Command Pallet` 
start typing `C# to plantUML`
Then run the first command that shows up.
that's should work

**Comment by VitoNguyen** [comment]
Hi,

I install these extensions: PlantUML, CSharp to PlantUML into VS Code. 
 - the Csharp2plantuml: Input Path = ./  
 - the Csharp2plantuml: Output Path = plantuml
 
I enter Ctrl + Shift + P and click on  C# to PlantUML: Class-diagram

But the nothing is happen and the directory 'plantuml' also is not created.

My Enviroment: Windows 10, Visual Studio Code 1.73.1 , PlantUML v2.17.4, CSharp to PlantUML v1.3.1 

How can i generate the UML class diagram ? Please give me your advice, many thanks for your help

Thanks & Best regards

Nguyen Xuan Viet
